src/NuGet.Clients/PackageManagement.VisualStudio/Strings.resx

```diff
@@ -195,4 +195,7 @@
   <data name="FailedToUpdateBindingRedirects" xml:space="preserve">
     <value>Failed to update binding redirects for {0} : {1}</value>
   </data>
+  <data name="ConfigErrorDialogBoxTitle" xml:space="preserve">
+    <value>Operation failed</value>
```

Comment:
How about ``NuGet operation failed``

It can be difficult to tell in customer screen shots where errors are coming from